modules/sharedaddy/sharing-service.php

```diff
@@ -726,10 +726,19 @@ function sharing_display( $text = '', $echo = false ) {
 		}
 	}
 
+	/**
+	 * Filters the content of the Sharedaddy sharing links
+	 *
+	 * @since 3.8.0
+	 *
+	 * @param string $sharing_content Content of the Sharedaddy sharing links
+	 */
+	$sharing_display = apply_filters( 'sharing_display', $sharing_content );
```

Comment:
See latest commit. I changed the filter name as you suggested.

On Thu, Oct 22, 2015 at 11:02 AM, Derek Smart <notifications@github.com>
wrote:

> In modules/sharedaddy/sharing-service.php
> <https://github.com/Automattic/jetpack/pull/2835#discussion_r42759381>:
>
> > @@ -726,10 +726,19 @@ function sharing_display( $text = '', $echo = false ) {
> >  		}
> >  	}
> >
> > +	/**
> > +	 * Filters the content of the Sharedaddy sharing links
> > +	 *
> > +	 * @since 3.8.0
> > +	 *
> > +	 * @param string $sharing_content Content of the Sharedaddy sharing links
> > +	 */
> > +	$sharing_display = apply_filters( 'sharing_display', $sharing_content );
>
> let's name this something more specific like
> jetpack_sharing_display_markup
>
> h/t @jeherve <https://github.com/jeherve>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Automattic/jetpack/pull/2835/files#r42759381>.
>



-- 
badfeather.com